dataprofiler/tests/profilers/test_data_labeler_column_profile.py

```diff
@@ -391,3 +395,172 @@ def test_empty_data(self, *mocks):
 
         diff_profile = profiler1.diff(profiler2)
         self.assertIsNone(merge_profile.data_label)
+
+
+class TestDataLabelerColumnProfilerNoMock(unittest.TestCase):
+    def test_json_encode(self):
+        profiler = DataLabelerColumn("")
+
+        serialized = json.dumps(profiler, cls=ProfileEncoder)
+
+        expected = json.dumps(
+            {
+                "class": "DataLabelerColumn",
+                "data": {
+                    "name": "",
+                    "col_index": float("nan"),
+                    "sample_size": 0,
+                    "metadata": {},
+                    "times": {},
+                    "thread_safe": False,
+                    "_max_sample_size": 1000,
+                    "data_labeler": "structured_model",
```

Comment:
here's a q. do we want to use `data_labeler` or do we want to do something sneaky like:
`load_data_labeler_options` so it wouldn't get directly loaded accidentally, but can be identified and load the object to go in as the `data_labeler` when deserialized.

Reply:
another option is to change this to a dictionary where the key is `from_library`... which will allow us to do `from_disk` down the road if someone brings their own model. 


```
"data_labeler": {"from_library": "structured_model"}
```


down the road we could do 
```
"data_labeler": {"from_disk": "my_model/path.json"}
```